packages/medusa/src/loaders/config.ts

```diff
@@ -102,8 +102,8 @@ export default (rootDirectory: string): ConfigModule => {
     modules: configModule.modules ?? {},
     featureFlags: configModule?.featureFlags ?? {},
     plugins: configModule?.plugins ?? [],
-    directories: configModule?.directories ?? {
-      srcDir: "dist",
+    directories: {
+      srcDir: process[Symbol.for("ts-node.register.instance")] ? "src" : "dist",
```

Comment:
is this really needed if its not used in a lot of places and not configurable?

Reply:
We are curretly discussing it in slack :D

Reply:
Moved it to the place where it belongs. No longer in the config